### PR TITLE
Run CI using GitHub actions

### PR DESCRIPTION
**Pull request by nullobject**
This PR updates the project to run CI using GitHub actions, instead of Travis CI.

It adds workflows for both running tests and publishing a package to NPM when a new release is tagged. I also added an [organisation secret](https://github.com/organizations/conversation/settings/secrets/actions) for the NPM token.

See also:

- https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
- https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages

## How to review

* Check that CI is running in workflows
* I'm not sure how we can test the publishing workflow, we might have to try it and adjust? I'm open to ideas.

### CODE REVIEW

**Comment by nickbrowne** [comment]
> * I'm not sure how we can test the publishing workflow, we might have to try it and adjust? I'm open to ideas.

Perhaps try release a test version tagged as beta? I think it's just `npm publish --tag beta`

**Comment by nullobject** [comment]
@nickbrowne Ah, good idea. Will do.

**Comment by jameshill** [comment]
Hey @nullobject awesome work can you update the README to also reflect these changes:

https://github.com/conversation/ui/blob/eb10dc2949a8d783e2e3ecae15050c7800be8a6b/README.md?plain=1#L115-L12

Screen Shot 2021-11-24 at 10 34 44 am
7

**Comment by nullobject** [comment]
@nickbrowne @jameshill The README has been updated.

After a bit of fiddling around, I created a beta release just to check everything works. You can see that it was successfully published to NPM: https://www.npmjs.com/package/@theconversation/ui/v/2.38.1-beta.0

File: README.md [hunk 1]
```diff
@@ -114,17 +114,23 @@ To run the example react app:
...
 
 ```sh
 > npm version minor
 > git push --follow-tags
 ```
 
+You can then create a release from the tag either using the web UI, or from the CLI:
+
+```sh
+> gh release create v1.2.3
```
Comment (jameshill) [inline README.md:131]: nice

**Review by jameshill — APPROVED** [review]
Good from me... 

@nickbrowne over to you.

**Comment by nickbrowne** [comment]
Yep everything looks sensible to me :+1:

**Pull request merged**